src/preset_cli/cli/superset/sync/dbt/command.py

```diff
@@ -112,25 +119,24 @@ def dbt_core(  # pylint: disable=too-many-arguments, too-many-branches, too-many
     preserve_columns: bool = False,
     preserve_metadata: bool = False,
     merge_metadata: bool = False,
+    raise_failures: bool = False,
 ) -> None:
     """
     Sync models/metrics from dbt Core to Superset and charts/dashboards to dbt exposures.
     """
     auth = ctx.obj["AUTH"]
     url = URL(ctx.obj["INSTANCE"])
     client = SupersetClient(url, auth)
+    deprecation_notice: bool = False
+    if raise_failures:
+        warnings.simplefilter("always", DeprecationWarning)
```

Comment:
I added this otherwise the `DeprecationWarning` doesn't get logged in the terminal.